joss/paper.md

```diff
@@ -0,0 +1,88 @@
+
+Here we highlight some of the simulation functionality available:
+
+***Age-stratified hospitalisation and death risks***: the outbreak simulation has three risks: hospitalisation risk (`hosp_risk` argument), death risk in hospital (`hosp_death_risk`) and death risk outside of hospital (`non_hosp_death_risk`), which all, by default accept a single number to define the population risk. In many infectious disease outbreaks risk is non-uniform across ages, so simulist accepts risks stratified by age using a user-defined `data.frame`, so any age-stratification is accepted. See the [Age-stratified hospitalisation and death risks vignette](https://epiverse-trace.github.io/simulist/index.html) for a complete explanation.
+
+***Age-structured population***: the outbreak simulation in simulist assumes a uniform population age between a lower and upper age range (0 and 90 by default). However, in order to accurately generate synthetic data for outbreak scenarios where the population age structure is known this can be input into the outbreak simulation, again using a user-defined `data.frame` to flexibility specify an arbitrary number of age groups and proportions. See the [Age-structured population vignette](https://epiverse-trace.github.io/simulist/dev/articles/age-struct-pop.html) for more details and a worked example.
+
+***Time-varying case fatality risk***: In addition to the ability to age-stratify hospitalisation and death risks, as outlined above, it is also possible to vary the fatality risk through time in the outbreak simulation. This is setup to accept a user-defined R function that takes the baseline risk and the simulation time ($t$) and calculates the fatality risk at time $t$. This is again setup in such a way as to maximise flexibility to the users and allows continuously varying (monotonic and non-monotonic functions)or discrete step-wise functions. The aim of this functionality is to phenomenologically mimic vaccination or non-pharmaceutical interventions without mechanistically modelling these in the simulation model. See the [Time-varying case fatality risk vignette](https://epiverse-trace.github.io/simulist/dev/articles/time-varying-cfr.html) for more explanation and examples.
+
+***Simulating empirical outbreak data***
+
+The outbreak simulation produces precise, standardised, and complete data, hereafter referred to as _ideal_ data. However, in reality those working with line list data often encounter a myriad of issues where the data they have available differs from the _ideal_ data. In simulist we offer two post-processing functions that modify _ideal_ line list data output by `sim_linelist()`to include two empirical characteristics, 1) real-time outbreak data and right-truncation (`truncate_linelist()`) and 2) inconsistent, missing, and duplicated data, so-called _messy_ data (`messy_linelist()`).
```

Comment:
What does precise mean in this context – usually means lack of uncertainty?

Reply:
Good point, I'll change to "exact", which I think better conveys what I mean.

Reply:
Resolved in bba9cb7b56d2e430583bd84f5ca9579a9a9a3027.